webui/react/src/components/WebhookCreateModal.tsx

```diff
@@ -132,6 +157,25 @@ const WebhookCreateModalComponent: React.FC<Props> = ({ onSuccess }: Props) => {
         id={idPrefix + FORM_ID}
         layout="vertical"
         onFieldsChange={onChange}>
+        {f_webhook && (
+          <>
+            <Form.Item label="Workspace" name="workspaceId">
```

Comment:
nit: this says it's required but doesn't seem to be in code. i see that there's a default value but you also allow the select to be cleared